README.md

```diff
@@ -111,6 +112,17 @@ DEEPGRAM_API_KEY = 'YOUR_API_KEY'
 deepgram = Deepgram(DEEPGRAM_API_KEY)
 ```
 
+## Custom API Endpoint
+
+In order to point the SDK at a different API endpoint (e.g., for on-prem deployments), you can pass in an object setting the `api_url` when initializing the Deepgram client.
```

Comment:
I would say "environment" as an endpoint would be like changing `v1/listen`. An environment would be like staging or an on-prem environment

```suggestion
In order to point the SDK at a different API environment (e.g., for on-prem deployments), you can pass in an object setting the `api_url` when initializing the Deepgram client.
```